specification/resources/monitoring/examples/curl/create_alert.yml

```
@@ -0,0 +1,7 @@
+lang: cURL
+source: |-
+  curl -X POST \
+    -H "Content-Type: application/json" \
+    -H "Authorization: Bearer $DIGITALOCEAN_TOKEN" \
+    "https://api.digitalocean.com/v2/monitoring/alerts"
```

This is missing the backslash after the URL value

```suggestion
    "https://api.digitalocean.com/v2/monitoring/alerts" \
```